### DISCUSSION TITLE
mini.pick - grep_live on current buffer

### DISCUSSION THREAD

**Discussion by diego-velez** [nvim-mini/mini.nvim#1418] [Q&A] [answered]
Is there a way to grep_live on my current buffer?

**Comment by diego-velez** [comment]
I'm trying:

```
    vim.keymap.set('n', '<leader>/', function()
      require(mini.pick).builtin.grep_live {
        globs = {
          vim.fn.expand '%F%m%r',
        },
      }
    end)
```

but it doesn't work for help files for example :(

**Comment by echasnovski** [comment, marked as answer]
My suggestion would be to use [`buf_lines` picker from 'mini.extra'](https://github.com/echasnovski/mini.nvim/blob/4228f166ee9db3e910eea1915e2d3683028add47/doc/mini-extra.txt#L231). You can use it like this: `:Pick buf_lines scope='current'`.
It won't use `rg` instead relying on default 'mini.pick' matching, but I'd argue `rg` is not needed here.

**Reply by diego-velez** [reply]
What a chad 😎, thank you very much!!
